lib/origami-build-tools-cli.js

```diff
@@ -179,10 +182,10 @@ if (task && (argument === 'develop' || argument === 'dev')) {
 				childProcess.kill();
 			}
 
-			childProcess = runWatchedTask(['demo']);
+			childProcess = runWatchedTask(devArguments);
```

Comment:
is it possible to have a test for this?

Reply:
I opened an issue. We should add tests for the whole command https://github.com/Financial-Times/origami-build-tools/issues/778